### DISCUSSION TITLE
"A client-side exception has occurred, see here for more info"  even thought _error exists

### DISCUSSION THREAD

**Discussion by walt1992** [vercel/next.js#41184] [Help]
Hello. I have _error.tsx to show server side error more informative to users. 

and I have expected  I must get an error page in server side with error messages with 500 error.

But, There exist another client side error as well. 

<img width="823" alt="image" src="https://user-images.githubusercontent.com/37332261/194083738-e156c14d-d3bd-43bf-b7f0-811f8828a91f.png">

and This error is same with the error from server side.  It can show full error message when I run dev mode. 

<img width="756" alt="image" src="https://user-images.githubusercontent.com/37332261/194084990-9d70abb6-23c1-4107-b390-6955eb2cbe57.png">

In this example, It is wierd that I got not only 500, but also 404 error(that is come from an api call from server side) as well.

It should be 500 error only logged in my client, right?

I think this is related with [this issue](https://github.com/vercel/next.js/issues/38609)

According to this issue, It already applied on canary on last Aug. So I guessed this is reloved in lastest version. 
But this is still occur.

**Comment by icyJoseph** [comment]
Sometimes this happens if you have an error on your `_error` page, and the surrounding code, for example `_app`. Say that the app crashes because a query was missing, then you try to render the error page, but the query checking logic is in `_app` so it crashes again while rendering the `_error`. Could you try to discard that scenario?
